.changeset/weak-lies-relax.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query-builder': minor
+---
+
+Refactor milestoning code
```

Comment:
Dont think we need a changeset for the refactor bit but we probably do need one for the confirmation message.